sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.institutions.controller.createApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -0,0 +1,417 @@
+  saveFakeApplicationDisbursements,
+  createFakeEducationProgramOffering,
+  createFakeUser,
+} from "@sims/test-utils";
+import {
+  ApplicationOfferingChangeRequestStatus,
+  ApplicationStatus,
+  InstitutionLocation,
+  OfferingIntensity,
+} from "@sims/sims-db";
+import { createFakeSINValidation } from "@sims/test-utils/factories/sin-validation";
+import { addDays } from "@sims/utilities";
+
+describe("ApplicationOfferingChangeRequestInstitutionsController(e2e)-createApplicationOfferingChangeRequest", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let collegeFLocation: InstitutionLocation;
+  let institutionUserToken: string;
+  let endpoint: string;
+  const OLD_ENV = process.env;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    // College F.
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFLocation = createFakeInstitutionLocation({ institution: collegeF });
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocation,
+    );
+    // Institution token.
+    institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+
+    endpoint = `/institutions/location/${collegeFLocation.id}/application-offering-change-request`;
+
+    // To check the study overlap, the BYPASS_APPLICATION_SUBMIT_VALIDATIONS needs to be set false.
+    process.env = {
+      ...OLD_ENV,
+      BYPASS_APPLICATION_SUBMIT_VALIDATIONS: "false",
+    };
+  });
+
+  it("Should able to submit application offering request for an eligible application by an institution.", async () => {
+    // Arrange
+    const savedUser = await db.user.save(createFakeUser());
+    // Student has a completed application to the institution.
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      { institutionLocation: collegeFLocation },
+      {
+        applicationStatus: ApplicationStatus.Completed,
+      },
+    );
+    // Student SIN Validation.
+    application.student.sinValidation = createFakeSINValidation({
+      student: application.student,
+    });
+    application.data.howWillYouBeAttendingTheProgram =
+      OfferingIntensity.fullTime;
+
+    await db.student.save(application.student);
+    await db.application.save(application);
+
+    // New offering with full time intensity.
+    const fakeOffering = createFakeEducationProgramOffering({
+      auditUser: savedUser,
+      institutionLocation: collegeFLocation,
+    });
+    // Updating study period, that belongs to the program year.
+    fakeOffering.studyStartDate = addDays(
+      5,
+      application.programYear.startDate,
+    ).toISOString();
+    fakeOffering.studyEndDate = addDays(
+      85,
```

Comment:
We can use the constant `OFFERING_STUDY_PERIOD_MIN_DAYS_FULL_TIME` for this.

Reply:
it has nothing to do with `OFFERING_STUDY_PERIOD_MIN_DAYS_FULL_TIME`, its just a random number